### ISSUE TITLE
.navbar-form has extraneous border-bottom for thinner windows

### ISSUE THREAD

**Issue by dbkaplun** [twbs/bootstrap#18606]
[jsbin](http://jsbin.com/hapiwoyono/1/edit?html,output)

![screenshot](http://i.imgur.com/WROwlrT.png)

**Comment by twbs-lmvtfy** [comment]
Hi @dbkaplun!

You appear to have posted a live example (http://jsbin.com/fewedozuwe/1/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 7, column 5 thru column 288: Attribute ```` integrity ```` not allowed on element ```` link ```` at this point.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by dbkaplun** [comment]
(fixed validation: [jsbin](http://output.jsbin.com/zolaqigeri/1), [validator](https://validator.w3.org/nu/?doc=http%3A%2F%2Foutput.jsbin.com%2Fzolaqigeri%2F1))

**Comment by twbs-lmvtfy** [comment]
Hi @dbkaplun!

You appear to have posted a live example (http://jsbin.com/zolaqigeri/1/edit), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 18, column 15: [E042](https://github.com/twbs/bootlint/wiki/E042): `.form-control` cannot be used on non-textual `<input>`s, such as those whose `type` is: `file`, `checkbox`, `radio`, `range`, `button`
* [W002](https://github.com/twbs/bootlint/wiki/W002): `<head>` is missing X-UA-Compatible `<meta>` tag that disables old IE compatibility modes
* [W003](https://github.com/twbs/bootlint/wiki/W003): `<head>` is missing viewport `<meta>` tag that enables responsiveness
* line 23, column 13: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.
* line 24, column 13: [W007](https://github.com/twbs/bootlint/wiki/W007): Found one or more `<button>`s missing a `type` attribute.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by dbkaplun** [comment]
[fixed jsbin hopefully](http://jsbin.com/hapiwoyono/1/edit?html,output)

**Comment by twbs-lmvtfy** [comment]
Hi @dbkaplun!

You appear to have posted a live example (http://jsbin.com/hapiwoyono/1/edit), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 23, column 15 thru column 84: Duplicate attribute ```` type ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by dbkaplun** [comment]
[fixed jsbin again](http://jsbin.com/venuyoveyu/1/edit?html,output)

**Comment by bassjobsen** [comment]
the first border is the `border-bottom` of the `.navbar-form` (https://github.com/twbs/bootstrap/blob/master/less/navbar.less#L290, https://github.com/twbs/bootstrap/blob/master/less/navbar.less#L453) and the second border is the `border` of the `.navbar .navbar-default`. (https://github.com/twbs/bootstrap/blob/master/less/navbar.less#L15, https://github.com/twbs/bootstrap/blob/master/less/navbar.less#L397)

Possible fix for your issue:

        .navbar-default .navbar-form {
            border-bottom: 0;
        }

**Comment by dbkaplun** [comment]
My particular example is easy to fix, yes, but I filed the bug because it seems like this is undesired behavior for Bootstrap. Maybe something like this would fix the underlying issue:
```css
.navbar-default :last-child {
  border-bottom: 0;
}
```

**Comment by bassjobsen** [comment]
@dbkaplun also your fix only fix your issue because of only the `.nav-form` got a top- and bottom- border, ( `.nav-text` has no border)

**Comment by mdo** [comment]
[**Bootstrap 3 is no longer being officially developed or supported.**](https://github.com/twbs/bootstrap/issues/20631)

All work has moved onto our next major release, v4. As such, this issue or pull request is being closed as a "won't fix." For additional help and support, we recommend [utilizing our community resources](https://github.com/twbs/bootstrap/blob/master/README.md#community). Thanks for your understanding, and see you on the other side of v4!

<3,
@mdo and team
